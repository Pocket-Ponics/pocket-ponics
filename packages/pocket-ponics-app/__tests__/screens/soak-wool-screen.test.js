import React from 'react'
import { shallow } from 'enzyme'

import SoakWoolScreen from '../../screens/soak-wool-screen'

describe('Soak Wool Screen', () => {
	test('renders correctly', () => {
		const tree = shallow(
			<SoakWoolScreen />
		).html()
		expect(tree).toMatchSnapshot()
	})
})