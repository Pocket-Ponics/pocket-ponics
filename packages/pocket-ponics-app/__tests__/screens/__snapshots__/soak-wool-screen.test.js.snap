// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Soak Wool Screen renders correctly 1`] = `"<View style=\\"flex:1\\"><View style=\\"background-color:#472600;color:#FFFFFF;padding:20px;flex-direction:column;flex:1\\"><Text style=\\"color:#FFFFFF;font-size:32px;margin-bottom:20px;text-align:center\\">Soak the rockwool</Text><Text style=\\"color:#FFFFFF;font-size:18px;margin-bottom:20px\\">Soak the included rockwool in water for 5 minutes, and then remove the rockwool from the water.  Shake off excess water, and place the rockwool on the tray.</Text><Image source=\\"1\\" style=\\"resize-mode:contain;width:100%;flex:1;margin-bottom:20px\\"></Image><View style=\\"background-color:#456E31;border-radius:5px;padding:10px;margin:10px;align-items:center;opacity:1\\"><Text style=\\"color:#FFFFFF;font-size:22px\\">Continue Setup</Text></View><View style=\\"align-items:center;opacity:1\\"><Text style=\\"color:#84AB71;font-size:22px\\">Cancel Setup</Text></View></View></View>"`;
